### PR TITLE
Create 2019-12-29-countdown-to-the-new-year-ember-code-snippet.md

### LINKED ISSUE #447: Create a blog post for December 29, 2019 about the `ember-code-snippet` addon
## Goal
Write a short blog post for the December blog series that is similar to the [example post]( https://github.com/ember-learn/ember-blog/blob/be8f29ba0ae8b5367d1758b1a150c1b503812a77/source/2019-12-01-countdown-to-the-new-year-ember-sortable.md). You can find more information on how to write your post in our [Contribution guidelines for DecEmber](https://github.com/ember-learn/ember-blog/blob/master/source/december-2019-blog-series-template.md).

## Criteria

- [ ] file name should 2019-12-DD-countdown-to-the-new-year-ADDON-NAME.md where DD is replaced with the day and ADDON-NAME is replaced with the hyphenated name of the addon
- [ ] The opening sentence should be updated where COUNT is the sequential ordinal word. 

```
**This is the COUNT in our DecEmber series- "Countdown to The New Year: 31 Days of 
Ember Addons".  We plan to highlight a new addon each day until the new year, and 
we hope you'll join us for the fun!**
```

- [ ] The first h2 should be "Day NUM" where NUM is the same as the DD in the date.
- [ ] the link to the addon should point to the page on Ember Observer
- [ ] There should be two sub sections: (both h3)- "What it does" and "Why I like it" 
- [ ] Include whether or not the addon has been updated yet for Ember Octane- if not, invite contributors to help get the addon Octane ready.
- [ ] The closing sentence can be slightly altered but should feel similar: 

```
"Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy!"
```

### PR DESCRIPTION
**Pull request by ijlee2**
## What it does

This creates a new blog post for the DecEmber 2019 blog series (closes #447). Please merge this on December 29th, 2019.

I added screenshots to illustrate how ember-code-snippet can be used with various libraries for syntax highlighting.

preview

### CODE REVIEW

File: source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+tags: 2019, Addons, DecEmber
+alias: 2019-12-29-countdown-to-the-new-year-ember-code-snippet.md
+responsive: true
+---
+
+**This is the twenty-ninth in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 29
+
+Before I took this assignment, I didn't know anything about [ember-code-snippet](https://emberobserver.com/addons/ember-code-snippet). Now, I'm glad that I did. This addon surfaced in 2015, has received regular updates, and powers many of my favorite addons. (Can you guess how?)
```
Comment (jayjayjpg) [inline source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md:13]: The personal note in the intro is really cool, imho :tada:

File: source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+READMORE
+
+### What It Does
+
+[ember-code-snippet](https://github.com/ef4/ember-code-snippet) lets you render code snippets in your app (or addon). Quite handy if you want to create a technical blog or **document your components**.
+
+It provides a helper called `get-code-snippet` so that (1) you can save code snippets in a dedicated folder and (2) create your own component to customize their look.
+
+To illustrate the use, we'll consider a code snippet from [Ember.js Octane vs Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/):
```
Comment (jayjayjpg) [inline source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md]: What do you think about the following?

```suggestion
To illustrate the use, we'll consider a code snippet from [the Ember.js Octane vs Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/):
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left one more small suggestion (feel free to apply or leave it whatever you find most useful) and I think the illustrations are great, too! Thank you for another :sparkles: writeup for December 🌈

File: source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md (hunk 1 above)
Comment (ijlee2) [reply source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md:13]: Thanks! I felt lucky to have worked on this.

File: source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md (hunk 2 above)
Comment (ijlee2) [reply source/2019-12-29-countdown-to-the-new-year-ember-code-snippet.md]: Yep, I took the title from their website, but think adding "the" helps the sentence flow better. Thanks!

**Review by MelSumner — APPROVED** [review]
Thank you for writing this! :+1:

**Pull request merged**